packages/terra-clinical-item-collection/tests/jest/Item.test.jsx

```diff
@@ -0,0 +1,335 @@
+import React from 'react';
+import Item from '../../src/Item';
+
+const listItemStyles = {
+  listItemLayout: 'twoColumns',
+  listItemTextEmphasis: 'start',
+  isListItemTruncated: true,
+  accessoryAlignment: 'alignTop',
+};
+
+it('should render a default Item', () => {
+  const item = (<Item />);
+  const component = shallow(item);
+  expect(component).toMatchSnapshot();
+});
+
+it('throws an error for a missing key when an item is selectable', () => {
+  try {
+    const item = (
+      <Item
+        startAccessory={<p>start</p>}
+        comment={<Item.Comment text="comment" />}
+        endAccessory={<p>end</p>}
+        isSelectable
+      >
+        <Item.Display text="Display 1" />
+      </Item>
+    );
+    shallow(item);
+  } catch (e) {
+    expect(e.message).toContain('Key is required for correct selectable implementation.');
```

Comment:
Is this catching the exception? I was thinking you need

```
throw new Error('oops');
```

as opposed to `console.error` to trigger this.